<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>489</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ChangeTheFileName</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButton_14">
     <property name="text">
      <string>Переименовать</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Старое имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Новое имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Результат</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Нажмите &quot;Переименовать&quot; что бы применить изинения</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Открыть папку</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Маска для имени файла</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>[N] Имя</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>[YMD] Дата</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>[C] Счетчик</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>[hms] время</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="text">
         <string>[N]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Настроить счетчик</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="3" column="0">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Количество цифр</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_3">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Начать с:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Шаг:</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>расширение</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>[E] Тип</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_13">
        <property name="text">
         <string>[C] Счетчик</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>[E]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Верхний/Нижний регист</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Без изменений</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>все строчные</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ВСЕ ЗАГЛАВНЫЕ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Первая буква заглавная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Начало Слов С Заглавной</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Поиск и переименование</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_10">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Заменяем</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ищем</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_15">
        <property name="text">
         <string>Замена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
